master/buildbot/changes/gerritchangesource.py

```diff
@@ -123,19 +151,23 @@ def flatten(properties, base, event):
 
         properties = {}
         flatten(properties, "event", event)
-        event_with_change = "change" in event and "patchSet" in event
+        properties["event.source"] = self.__class__.__name__
         func_name = "eventReceived_%s" % event["type"].replace("-", "_")
         func = getattr(self, func_name, None)
-        if func is None and event_with_change:
+        if func is None:
             return self.addChangeFromEvent(properties, event)
-        elif func is None:
-            if self.debug:
-                log.msg("unsupported event %s" % (event["type"],))
-            return defer.succeed(None)
         else:
```

Comment:
you do not need this `else:` after the `return` statement

Reply:
Done